### DISCUSSION TITLE
Cobbler Ansible inventory plugin - pulling IP addresses from cobbler

### DISCUSSION THREAD

**Discussion by opoplawski** [cobbler/cobbler#3451] [General]
I'm working on an enhancement to the cobbler ansible inventory plugin here:

https://github.com/ansible-collections/community.general/pull/6711

that would add an `ip_address` and/or `ipv6_address` host variable with the
primary IP address of the host in cobbler as well as add `cobbler_ip_addresses` and `cobber_ipv6_addresses` dicts that contain DNS name to IP address mappings for all of the defined interfaces in cobbler. 

Looking for some feedback - does this seem like a useful format / naming?  Any other requests for the inventory plugin?

**Comment by SchoolGuy** [comment]
I think it is very reasonable. Looking at the provided PR I think the problem with the IP address of the Cobbler server can be solved with naming it:

- `cobbler_server_ipv4_addresses`
- `cobbler_server_ipv6_addresses`

I would also propose that we name the IPv4 address field `cobbler_ipv4_addresses` since an IP can be more than one protocol and as such I would like to avoid ambiguous naming.

**Comment by opoplawski** [comment]
Thanks for the feedback - I've made those changes.

**Comment by SchoolGuy** [comment]
@Algirdas-Z is using this in production. Do you have an opinion?

**Reply by Algirdas-Z** [reply]
Hey, my scenario changed, so i do not use it at the moment, however proposed changes seems to me very useful.
